modules/extensions/process/publisher/process/themes/default/partials/add_doc.hbs

```diff
@@ -49,7 +49,7 @@
                                 <input type="hidden" id="docExtension" name="docExtension"/>
                             </div>
 
-                            <div class="span6" style="float:right;width:48%;">
+                            <div class="span6" style="float:right;width:48%;margin-top: 25px;">
```

Comment:
This is fixed by https://github.com/wso2/product-pc/commit/e931bb704e9065cf57f08f61619a45a4beecb5c7